packages/server/modules/workspaces/graph/resolvers/workspaces.ts

```diff
@@ -761,6 +761,12 @@ export = FF_WORKSPACES_MODULE_ENABLED
         },
         role: async (parent) => {
           return parent.workspaceRole
+        },
+        projectRoles: async (parent, _args, ctx) => {
+          return await ctx.loaders.workspaces!.getProjectRolesByWorkspaceId.load({
```

Comment:
Not sure if there's a point to this dataloader, what would be the scenario in which these calls could be batched together? Each entry is always gonna be its own workspace & user id pair and thus a separate query 🤔

Reply:
I just read the implementation and understand this better. If workspaceId is the same and multiple users can be batched together, then there is in fact a point to this dataloader. But yea, the API should be less misleading - the workspaceId cannot be different between dataloader load() requests